test/validations/relaxng/run.py

```diff
@@ -20,14 +21,26 @@
 relaxng_doc = etree.parse('test/validations/relaxng/schema.xml')
 relaxng = etree.RelaxNG(relaxng_doc)
 
+ruleset_whitelist = None
+
+with open('utils/ruleset-whitelist.csv', 'r') as ruleset_whitelist_file:
+    ruleset_whitelist_reader = csv.reader(ruleset_whitelist_file)
+    ruleset_whitelist = list(ruleset_whitelist_reader)
+
 exit_code = 0
 
 print("Validation of rulesets against relaxng schema.xml begins...")
 
 for filename in sorted(files):
+    basename = os.path.basename(filename)
+    skip_relaxng = False
+    for entry in ruleset_whitelist:
+        if entry[2] == '1' and entry[3] == basename:
```

Comment:
We can have a small performance gain if you do `if entry[3] == basename and entry[2] == '1':` insteads